saleor/extensions/manager.py

```diff
@@ -272,7 +272,11 @@ def list_payment_gateways(self) -> List[dict]:
             }
             for plugin in self.plugins
             if payment_method in type(plugin).__dict__
-            and self.get_plugin_configuration(plugin.PLUGIN_NAME).active
+            and (
```

Comment:
Maybe it would be a good idea to create a normal `if` statement instead of a conditional assignment .  The amount of condition is at least here, we could increase readability. What do you think? @szewczykmira